### ISSUE TITLE
Moduleverse down?

### ISSUE THREAD

**Issue by jwjames83** [OutbreakInc/Galago#15]
I'm trying to initialize a project and it's stuck.

node ....\sdk\bin\SDK.js --init
Did not find an SDK on your system.  I will try to download the latest one
 and then build your project.
Note: you must have an internet connection to update.

**Comment by kuym** [comment]
Yes, sadly Moduleverse is no more. I'm looking at a way to allow continued support of the Galago platform without requiring the proprietary infrastructure that supported it.  In the three years since I created the Galago SDK I've applied aspects of it to a number of commercial (and proprietary) products and substituted in components - like the gcc toolchain - from other sources.  I will look at reworking or replacing the SDK.js script to use the excellent [Pebble embedded ARM SDK](http://developer.getpebble.com/sdk/install/) which is Mac/Linux/Windows and a drop-in replacement for that part of the Galago build chain.

In the mean time, let me get back to you asap about how you can work around this issue.

**Comment by jwjames83** [comment]
Thanks for the quick reply, but take your time - I'm in no rush to get it going. I just tinker with it every so often.
